src/data/sql-quiz.ts

```diff
@@ -1,4 +1,57 @@
 const sqlQuiz = [
+  {
+    Question: "In SQL,How many Primary keys can be created on a table?",
```

Comment:
removing unnecessary capitalization

```suggestion
    Question: "In SQL, how many primary keys can be created in a table?",
```